bin/run_electrometer.py

```diff
@@ -1,7 +1,5 @@
+#!/bin/env python
 import asyncio
 from lsst.ts.electrometer.csc import ElectrometerCsc
```

Comment:
Usually I am asked to order the imports such that standard modules and of non-standard imports are separated by a blank line. I'll leave it to you to decide whether you want to reorder the imports and introduce a blank line or not.